.gitignore

```diff
@@ -15,5 +15,5 @@ cmake-build-debug
 deps
 ebin
 log
-
-
+rebar.lock
```

Comment:
I'm not sure we should ignore `rebar.lock` given that it overrides `rebar.config` unless specifically upgraded, I could see this causing some unintentional problems

Reply:
rebar specifically recommends not to ignore the rebar.lock file, good catch